### PR TITLE
Fix id67 RGB Matrix

### PR DESCRIPTION
**Pull request by waffle87**
## Description

`ID67_CAPS_LOCK_KEY_INDEX` is never defined anywhere. I assume its correct value is 23...

## Types of Changes

- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Review by fauxpark — APPROVED** [review]

File: keyboards/idobao/id67/id67.c [hunk 1]
```diff
@@ -56,6 +56,6 @@ led_config_t g_led_config = { {
 __attribute__ ((weak))
 void rgb_matrix_indicators_user(void) {
```
Comment (vinorodrigues) [inline keyboards/idobao/id67/id67.c:57]: this entire function should be removed
1. caps lock state should not be coded into the core file.  also #FFFFFF is really bright !  This value should be based on current brightness values
2. value would have been 36.  idx-2-phys & led-idx are reversed, but not the kmtrx-2-led-idx in `g_led_config`

File: keyboards/idobao/id67/keymaps/vinorodrigues/keymap.c [hunk 2]
```diff
@@ -121,15 +121,15 @@ void rgb_matrix_indicators_advanced_user(uint8_t led_min, uint8_t led_max) {
 
     if (host_keyboard_led_state().caps_lock) {
         if (isRGBOff) {
-            rgb_matrix_set_color(ID67_CAPS_LOCK_KEY_INDEX, v, v, v);  // white
```
Comment (vinorodrigues) [inline keyboards/idobao/id67/keymaps/vinorodrigues/keymap.c:124]: line 85 of this file should contain:
```
#define CAPS_LOCK_KEY_INDEX 36  // position of Caps Lock key
```

this value is specific to this keymap ... not sure how it ended up in root file `id67.c`

**Review by tzarc — APPROVED** [review]

**Review by vinorodrigues — CHANGES_REQUESTED** [review]

**Comment by tzarc** [comment]
Well that was poorly timed... guess a followup PR is required.

**Comment by vinorodrigues** [comment]
ok ... i'll do it.  gimme a sec

**Comment by vinorodrigues** [comment]
ok, done.  #16917

thanks @tzarc

File: keyboards/idobao/id67/keymaps/vinorodrigues/keymap.c (hunk 2 above)
Comment (waffle87) [reply keyboards/idobao/id67/keymaps/vinorodrigues/keymap.c:124]: Ah, sorry. I suppose I was thrown off by this comment 
[https://github.com/qmk/qmk_firmware/blob/master/keyboards/idobao/id67/id67.c#L22](https://github.com/qmk/qmk_firmware/blob/master/keyboards/idobao/id67/id67.c#L22)

Comment (vinorodrigues) [reply keyboards/idobao/id67/keymaps/vinorodrigues/keymap.c:124]: np.  it's confusing as heck.  i had to retest my assumptions twice to confirm what i'd written a few months back.

and hey!  thanks for trying to fix it - was was slow to see that it was posted and didn't check anyway.  by the time i realised you'd already pr'ed it.   no foul and the community thanks you for your contributions!

**Pull request merged**